+nosnoc/+activeset/Cls.m

```diff
@@ -0,0 +1,35 @@
+classdef Cls < handle
+    properties(SetAccess=private)
+        active_constraints
```

Comment:
the names of properties in different classes have different style, compare to heaviside, please make consistent.

Reply:
They are named differently because they _are_ different, in my opinion. I.e. in PDS/CLS it is active unilateral constraints that define the active set, wheras in PSS it is specifically the active regions, and in Heaviside case it is which of the three branches of the Heaviside step each step function is on.

I am happy to change the names but making them similar is wrong in my opinion.

Reply:
Okay. I agree.